flax/nn/attention.py

```diff
@@ -166,7 +168,9 @@ def scan_in_dim(*args, **kwargs):
 
 
 class Cache(Collection):
-  """Collect intermediate activations for efficient autoregressive decoding."""
+  """DEPRECATION WARNING:
+  `flax.nn` is deprecated, use `flax.linen` instead.
```

Comment:
Please add the link to the README, like I suggested above, to these deprecation warnings as well.